tilequeue/wof.py

```diff
@@ -330,10 +332,25 @@ def failure(reason):
     cessation = min(edtf_cessation.latest_date(),
                     edtf_deprecated.latest_date())
 
+    # grab any names in other languages
+    lang_suffix_size = len('_preferred') * -1
+    l10n_names = {}
+    for k, v in props.iteritems():
+        if not v:
+            continue
+        if not k.startswith('name:') or not k.endswith('_preferred'):
+            continue
+        if isinstance(v, list):
+            v = v[0]
+        lang = k[:lang_suffix_size]
```

Comment:
Doesn't this take the _first_ `lang_suffix_size` chars, rather than drop the last chars?